src/main/kotlin/org/jitsi/rtp/rtcp/RtcpByePacket.kt

```diff
@@ -39,6 +40,7 @@ import org.jitsi.rtp.extensions.unsigned.toPositiveLong
  * (opt) |     length    |               reason for leaving            ...
  *       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
  */
+@SuppressFBWarnings("NP_NULL_ON_SOME_PATH")
```

Comment:
Reading about this warning it seems bad, why can we suppress it?

Reply:
I could not figure out what the path it discovered is. I think it's a false positive due to some assumptions that kotlin makes that aren't present in the bytecode. I thought about disabling the warning for all kotlin code since it has null safety already, but I figured if it's just one instance I'll keep it and see what happens.

Reply:
Hm, ok. Maybe worth a comment in the code...it's a bummer the rule names are so cryptic.

Reply:
This code is enough to trigger it:
```
class Test {
    val reason: String? by lazy { "non-null" }
}
```

I'll supress it for all kotlin files.